package.json

```diff
@@ -11,13 +11,24 @@
     "start": "cross-env NODE_ENV=development tsc-watch --onSuccess \"node .\"",
     "prebuild": "npm-run-all clean lint",
     "build": "tsc",
+    "build:no-check": "tsc",
     "prestart": "npm run build",
     "lint": "eslint . --ext .ts --fix",
     "format": "prettier --write src/**/*.ts",
     "test": "jest",
+    "pretypeorm": "npm-run-all clean build:no-check",
     "typeorm": "node -r module-alias/register ./node_modules/typeorm/cli.js",
-    "typeorm:sync": "node -r module-alias/register ./node_modules/typeorm/cli.js schema:sync",
-    "pretypeorm:sync": "npm-run-all clean build"
+    "typeorm:sync": "npm run typeorm schema:sync",
+    "migration:gen": "npm run typeorm migration:generate --",
```

Comment:
What's the difference between generate and create? Just curious : p

Reply:
gen looks at entities to generate sql that describes the diff, create creates empty